setup.py

```diff
@@ -33,27 +33,19 @@
     license='License :: OSI Approved :: MIT License',
     packages=find_packages(),
     install_requires=[
-        'Sphinx',
-        'Jinja2',
-        'Pygments',
-        'docutils',
-        'kruxstatsd',
-        'statsd',
-        'argparse',
-        'GitPython',
-        'simplejson',
-        'tornado',
-        'lockfile',
-        'pygerduty',
-        'async',
-        'fudge',
-        'gitdb',
-        'smmap',
-        'subprocess32',
+        'pystache==0.5.4',
+        'Sphinx==1.4.6',
+        'kruxstatsd==0.3.0',
+        'argparse==1.4.0',
+        'tornado==3.0.1',
+        'simplejson==3.8.2',
+        'GitPython==2.0.8',
+        'lockfile==0.12.2',
+        'subprocess32==3.2.7',
```

Comment:
Please do not specify versions in `setup.py`. See https://kruxdigital.jira.com/wiki/display/EN/How+to+set+up+transitive+dependencies+for+a+python+project for why this is a problem.